pkgs/applications/misc/snowsql/darwin.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  unzip,
+  cpio,
+  gzip,
+  unar,
+  darwin,
+  zlib,
+  version,
+  meta,
+  pname,
+}: let
+  arch =
+    if stdenv.isAarch64
+    then "arm64"
+    else "x86_64";
+in
+  stdenv.mkDerivation rec {
```

There is a preference to use `finalAttrs` rather than `rec`. Note that the newly opened parenthesis needs to be closed at the end of the package (see suggestion).
```suggestion
  stdenvNoCC.mkDerivation (finalAttrs: {
```